agrirouter-sdk-dotnet-standard-test/Service/Common/OnboardResponseSerializationTest.cs

```diff
@@ -0,0 +1,99 @@
+using Agrirouter.Api.Dto.Onboard;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Agrirouter.Test.Service.Common;
+
+public class OnboardResponseSerializationTest
+{
+    const string UppercaseOnboardResponseJson = @"{
+  ""DeviceAlternateId"": ""deviceAlternateId"",
+  ""CapabilityAlternateId"": ""capabilityAlternateId"",
+  ""SensorAlternateId"": ""sensorAlternateId"",
+  ""connectionCriteria"": {
+    ""GatewayId"": ""gatewayId"",
+    ""Measures"": ""measures"",
+    ""Commands"": ""commands"",
+    ""Host"": ""host"",
+    ""Port"": ""port"",
+    ""ClientId"": ""clientId""
+  },
+  ""authentication"": {
```

Comment:
This is lowercase by intention, because it already serialized in lowercase before